docs/preview/02-Features/assertion.md

```diff
@@ -0,0 +1,210 @@
+# Test assertion
+The `Arcus.Testing.Assert` library is a highly reusable and independent library that contains dev-friendly test assertions on different types of output (XML, JSON...). The purpose of the library is to write readable and test-friendly assertions that show quickly what went wrong with the (XML, JSON...) output of a certain piece of functionality.
+
+## Installation
+Install this package to easily assert on different output contents:
+
+```shell
+PM> Install-Package -Name Arcus.Testing.Assert
+```
+
+## XML
+The library has an `AssertXml` class that exposes usable test assertions when dealing with XML outputs. The most popular one is comparing XML documents.
+
+```csharp
+using Arcus.Testing;
+
+string expected = "<root>...</root>";
+string actual = "<diff-root>...</diff-root>";
+
+AssertXml.Equal(expected, actual);
+// Arcus.Testing.EqualAssertionException
+// AssertXml.Equal failure: expected and actual XML documents do not match
+// Expected element tag name 'root' but was 'diff-root' at /root
+//
+// Expected:
+// <root>
+//      ...
+// </root>
+//
+// Actual:
+// <diff-root>
+//      ...
+// </diff-root>
+```
+
+💡 Currently, the input contents are trimmed in case if the input is too big to be shown in a humanly readable manner to the test output. In case of large files, it might be best to log those files (or parts that interest you) separately before using this test assertion.
+
+### Customization
+The test assertion also expose several options to tweak the behavior of the XML comparison.
+
+```csharp
+using Arcus.Testing;
+
+AssertXml.Equal(..., options =>
+{
+    // Adds one ore more local names of XML nodes that should be excluded from the XML comparison.
+    options.IgnoreNode("local-node-name");
+
+    // Sets the type of order which should be used when comparing XML attributes.
+    // REMARK: only the order of XML attributes can be set, XML elements are still compared by their contents.
+    // Default: Ignore.
+    options.Order = AssertXmlOrder.Include;
+
+    // Sets the maximum characters of the expected and actual inputs should be written to the test output.
```

Comment:
That sentence is not very clear, maybe a "that" or "which" is missing? 
// Sets the maximum characters of the expected and actual inputs _that_ should be written to the test output.

Reply:
Fixed this here: https://github.com/arcus-azure/arcus.testing/pull/110/commits/e4a5129f9f64bb0ae9d6be5f44ed1067c13915c3